src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+
+{{% notice note %}}
+This implementation is designed using the [AWS IoT device shadow service](https://docs.aws.amazon.com/iot/latest/developerguide/iot-device-shadows.html) through reserved MQTT shadow topics, and describes two implementations that use device shadows. Please refer to the [Designing MQTT Topic for AWS IoT Core](https://docs.aws.amazon.com/whitepapers/latest/designing-mqtt-topics-aws-iot-core/designing-mqtt-topics-aws-iot-core.html), specifically the _Applications on AWS_ section. This white paper provides additional alternative patterns that go beyond the scope of this implementation.
+{{% /notice %}}
+
+## Use Cases
+
+- Request or update device state configuration
+  - _I want to track and control smart home devices from my mobile application_
+  - _I want to remotely control household heaters by turning them on/off or by setting up desired temperature_ 
+- Request device actions based on some commands
+  - _I want to remotely unlock a door for a family visitor_ 
+  - _I want to remotely instruct a device for some action, basis of a command_
+
+## Reference Architecture
+
+This section describes architecture considerations for command and control using the AWS IoT Core device shadow
+ - When requesting update to device state configuration
+ - When requesting device actions based on commands  
+
+{{< tabs groupId="MQTT-arch">}}
+{{% tab name="Device Configuration" %}}
+
+# ![Track and control device state configuration using shadow ](architecture1.svg)
+
+
+- _AWS IoT Core_ is the MQTT message broker processing messages
+- _Device_ is the IoT thing to be controlled
+- _Client Application_ is the remote logic that issues commands, update device state configuration and consumes device telemetry data
+- _Device Shadow_ is the replica which makes a device state available to applications and other services
+- _ShadowTopicPrefix_ is an abbreviated form of the topic, referred to `$aws/things/device1/shadow`(classic shadow topic)
+
+1. The _Device_ establishes an MQTT connection to the _AWS IoT Core_ endpoint and then subscribes to the reserved [MQTT shadow topics](https://docs.aws.amazon.com/iot/latest/developerguide/reserved-topics.html#reserved-topics-shadow) to get and update shadow event operations. Moreover, instead of subscribing individually to each reserved shadow topic, wildcard(_#_) could be used for generalized subscription like _`ShadowTopicPrefix/get/#`_ , _`ShadowTopicPrefix/update/#`_.
+
+    _`ShadowTopicPrefix/get/accepted`_   
```

Comment:
these are not showing as an unordered list. Also, these is not continuity between the two topics in the previous sentence and the list of other topics that make up the generalized set of shadow topics.